packages/styles/scss/patterns/simplelongform/_simplelongform.scss

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin simplelongform { 
+  .#{$prefix}--simplelongform {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+      @include carbon--make-col-offset(4, 16);
+      @include carbon--make-col(7, 16);
+    }
+    &__title {
+      @include type-style('expressive-heading-05');
+    }
+    &__content {
+      @include type-style('body-long-02');
+    }
+    &__divider {
+      border-bottom: 1px solid $ui-04;
+      &__col {
+        @include carbon--make-col-offset(4, 16);
+        @include carbon--make-col(7, 16);
```

Comment:
You're treating your columns almost more like rows. You only need one column for this layout I think (unless I'm misunderstanding something).  This shouldn't be necessary.

Reply:
SimpleLongform has title, content, link & divider. title content and link behave the same on different viewports they can be in one col and divider behave difference and has to be in another col. I don't think we want to wrap each columns within row as we have layout only one element for entire wide of container.

Reply:
I'm seeing all the content is `(4, 4)` for `sm`. The divider hangs. content is 100% of the column width it looks like. Not specified, but it probably maxes out at `640px` This might only happen for like `30px` since the breakpoint is at `672px`, so I don't know. Might need to circle in @wonilsuhibm for that.

<img src="https://user-images.githubusercontent.com/3793636/67887161-cbe3c680-fb18-11e9-925f-585c63b99b25.png" width="300px" />

`md` the content is `(8, 8)`. divider overhangs. content is maxes out at `640px` but it's width is `90%`.
<img src="https://user-images.githubusercontent.com/3793636/67887775-e9fdf680-fb19-11e9-9504-91c208056c2d.png" width="300px" />

`lg` and what appears to be larger is `(8, 16)`. offset `(4, 16)`. The divider (I missed this the first time around) is offset `(4, 16)` too, but it extends to what looks like `(12, 16)`. It still overhangs. The content is still max width of `640px` but it's width is `90%`.

<img src="https://user-images.githubusercontent.com/3793636/67888202-b4a5d880-fb1a-11e9-8382-8bb46232ea4e.png" width="300px" />

See the comments on the container width for content.

<img src="https://user-images.githubusercontent.com/3793636/67888258-cc7d5c80-fb1a-11e9-9af2-34d395abadcd.png" width="300px" />
<img src="https://user-images.githubusercontent.com/3793636/67888274-d3a46a80-fb1a-11e9-8d5b-6984852c9d29.png" width="300px" />

Reply:
Did you get the latest specs ?
I am afraid you might be referencing wrong specs. Specs from your screenshoot is a bit different from the one I have.